Dockerfile

```diff
@@ -0,0 +1,50 @@
+##### General instruction: ######
+#docker build -f ./Dockerfile -t opera-adt/compass:latest . #To build the docker image
+#to run docker container in interactive mode: docker run -it --name COMPASS opera-adt/compass:latest /bin/bash #To run docker container in interactive mode:
+#
+#
+##### For MacOS with Apple silicon: ######
+# docker build --platform linux/amd64 -f ./Dockerfile -t opera-adt/compass:latest . #To build the docker image
+# docker run -it --platform linux/amd64 --name COMPASS opera-adt/compass:latest /bin/bash #To run docker container in interactive mode:
+#
+#
+##### Mounting options - example ######
+# docker run -it --platform linux/amd64 -v $HOME/Desktop/gslc:/home/compass_user/gslc --name COMPASS opera-adt/compass:latest /bin/bash
+#
+#
+
+FROM ubuntu:latest
+
+SHELL ["/bin/bash","-c"]
+
+LABEL author="OPERA ADT" \
+      description="Dockerfile for OPERA COMPASS" \
+      version="2022.0527"
+
+RUN apt-get -y update &&\
+    apt-get -y install curl git &&\
+    adduser --disabled-password compass_user
+
+USER compass_user 
+
+RUN cd $HOME &&\
+    curl -sSL https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -o miniconda.sh &&\
+    bash miniconda.sh -b -p ${HOME}/python/miniconda3
+ENV PATH="/home/compass_user/python/miniconda3/bin:${PATH}"
+ENV PROJ_LIB=/home/compass_user/python/miniconda3/share/proj
```

Comment:
These should be set automatically when you activate the environment, and that's a more reliable way to set up the environment in case there are other variables that need to be set for other packages.